deepchem/models/tests/test_layers_from_config.py

```diff
@@ -527,6 +521,18 @@ def test_gru():
   assert layer_copied.init == layer.init
 
 
+@pytest.mark.torch
+def test_torch_gru():
+  n_hidden = 100
+  init = 'xavier_uniform_'
+  layer = dc.models.layers.GatedRecurrentUnit(n_hidden, init)
+  config = layer.get_config()
```

Comment:
I believe you meant to import the layer from dc.models.torch_models.layers.GatedRecurrentUnit(n_hidden, init), as that is where the new ported layer has been added?

Reply:
Added this in test_layers.py appropriately